docs/_includes/segmented-buttons.html

```diff
@@ -0,0 +1,122 @@
+<div id="segmented-buttons">
+    {% include section-header.html name="segmented-buttons" version=page.versions.segmentedButton %}
+
+    <p>Segmented Buttons are non-form toggle buttons used in certain cases. For detailed requirements and when to use these over tabs, please visit the <a href="https://ebay.gitbook.io/mindpatterns/input/segmented-buttons">eBay MIND Pattern for Segmented Buttons</a>. </p>
+    <p>The selected toggle button is toggled by <span class="highlight">aria-current="true"</span> attribute.</p>
+    <p><strong>NOTE:</strong> JavaScript is required to handle segmented buttons currently selected button.</p>
+    <p></p>
+
+    <h3 id="button-default">Default Segmented buttons</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="segmented-buttons">
+                <ul class="segmented-buttons__items">
+                    <li class="segmented-buttons__item"><button type="button" class="btn"
```

Comment:
So, we are using the `btn` class here, but what if a developer tries to add some of the other button classes? I imagine most of those button classes should never used here. Given that, might it be better to create a  `segmented-buttons__btn` and mixin just the base button styles instead?

Reply:
I think it might be okay but there's a lot of duplicate code which isnt in a mixin. I went with this method because we did the same in split button
I also think if users put some other classes and break it that's on them. We provided the guidance on what classes to use. If they deviate, well tough luck! 😉

Reply:
I went ahead and changed this. There were a lot of other classes I added as well. I don't think it's needed but it wasn't too bad.